docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md

```diff
@@ -29,47 +29,40 @@ Steps
     % pip3 install openlineage-common
     ```
 
-2. Update the action_list key in your Validation Operator config.
+2. Update the action_list key in your Checkpoint config.
 
-    Add the ``OpenLineageValidationAction`` action to the ``action_list`` key of the ``ActionListValidationOperator`` config in your ``great_expectations.yml``.
+    Add the ``OpenLineageValidationAction`` action to the end of your Checkpoint's ``action_list``.
 
     ```yaml
-    validation_operators:
-      action_list_operator:
-        class_name: ActionListValidationOperator
-        action_list:
-        - name: openlineage
-          action:
-            class_name: OpenLineageValidationAction
-            module_name: openlineage.common.provider.great_expectations
-            openlineage_host: ${OPENLINEAGE_URL}
-            openlineage_apiKey: ${OPENLINEAGE_API_KEY}
-            job_name: ge_validation # This is user-definable
-            openlineage_namespace: ge_namespace # This is user-definable
+    name: taxi_data
+    module_name: great_expectations.checkpoint
+    class_name: Checkpoint
+    # ...
+    action_list:
+      - name: openlineage  # this name is user-definable
+        action:
+          class_name: OpenLineageValidationAction
+          module_name: openlineage.common.provider.great_expectations
+          openlineage_host: ${OPENLINEAGE_URL}
+          openlineage_apiKey: ${OPENLINEAGE_API_KEY}
+          job_name: ge_validation # This is user-definable
+          openlineage_namespace: ge_namespace # This is user-definable
```

Comment:
Ideally, this and any other code in this how-to guide would be pulled from a script that is under test and linked to (the GitHub version) at the end of the guide.  That may fall outside the scope of this PR, but it's a standard to keep in mind striving for when we're updating the code in docs to begin with.